topics/galaxy-interface/tutorials/workflow-posters/tutorial.md

```diff
@@ -0,0 +1,271 @@
+objectives:
+- Learn how to utilize external tools to get high resolution images of you Workflow
+- Learn how to compress high resolution Workflow images to share them in the web
+time_estimation: 30M
+key_points:
+- Print ready screenshots
+- Basic image compression
+contributors:
+- ElectronicBlueberry
+level: Introductory
+subtopic: workflows
+---
+
+Sometimes we want to share our workflows outside of Galaxy, like printing it on a poster.
+Making screenshots of Workflows looses a lot of detail, often to an degree where information is lost.
+
+There are third party tools which can help us in making high resolution screenshots that preserve all details
+and produce pictures which are suitable for printing and sharing.
+
+{% include _includes/cyoa-choices.html option1="Print" option2="Web" default="Print"
+       text="If you want to print your Galaxy Workflow on a poster, choose \"Print\", otherwise \"Web\" is the better option." %}
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Making a high-resolution Screenshot of a Galaxy Workflow
+
+<div class="Print" markdown="1">
+In this section we will learn on how to make an image of a workflow which is suitable for printing on a poster.
+</div>
+
+<div class="Web" markdown="1">
+In this section we will learn on how to make a screenshot of a workflow which preserves as much details as possible.
+</div>
+
+We will be using a feature built into the [Firefox](https://www.mozilla.org/firefox/) web browser to make screenshots larger than our screen.
+This only works in Firefox, so make sure you have it installed and are using it to follow the rest of this tutorial.
+
+## Hiding the Galaxy UI
+
+<div class="Print" markdown="1">
+For our Workflow poster, we will likely not want to print the rest of Galaxys user interface.
+</div>
+
+<div class="Web" markdown="1">
+For our screenshot, we may want to hide the rest of Galaxys user interface, so the focus is on the Workflow itself.
+</div>
+
+We could cut it out of the final image using image-manipulation tools, but this is slow and cumbersome, especially for high-res images.
+
+From Galaxy version 23.2 onwards there is a feature with which we can hide the Galaxy UI directly.
+It is meant to be used for embedding workflows, but can be accessed directly, which is perfect for purpose.
+
+> <hands-on-title>Hiding the Galaxy UI for a Workflow</hands-on-title>
+>
+> 1. **Opening the Workflow Preview**
+>    - Navigate to your Workflows list
+>    - Press "{% icon galaxy-eye %} View" on your desired Workflow
+>    - You should now see the preview page, which should look similar to this:
+>      ![Galaxys Workflow Preview page with the workflow being rendered next to an info-box](./images/workflow-preview.png)
+> 2. **Add the following to the URL in your browser:**
+>
+>    ```
+>    &embed=true&buttons=false&about=false&heading=false&minimap=false&zoom_controls=false
+>    ```
```

Comment:
This is fantastic and will work very well for lots of folks!

Would it be worth adding a link/button in the workflow preview page, that has the same effects? E.g. below "last update", something, maybe labelled "Print View" in a future version of Galaxy? It could help with discoverability.

Reply:
moved to https://github.com/galaxyproject/galaxy/issues/17127